en/docs/testing/test-rest-endpoints-via-the-openapi-console.md

```diff
@@ -0,0 +1,32 @@
+# Test REST Endpoints via the OpenAPI Console
+
+Devant offers an integrated OpenAPI Console to test REST endpoints for the Service components you create and deploy. Since Devant secures REST APIs with OAuth 2.0 authentication, the OpenAPI Console generates test keys to help you test your APIs.
+
+Follow these steps to test a REST endpoint using the OpenAPI Console:
+
+1. Go to the [Devant Console](https://console.devant.dev/) and log in.
+2. Select the project.
+3. In the **Integrations** pane, click on the component you want to test.
```

Comment:
```suggestion
3. In the **Integrations** pane, click on the integration you want to test.
```
In devant we use `integration` instead of `component`